### PR TITLE
Solved drag&drop problem: a lot of DragEnter/DragLeave events when dragging over HelixViewport3D and its children.

### PR DESCRIPTION
**Pull request by isukces**
Solved drag&drop problem mentioned http://forum.helix-toolkit.org/topic/509250-drag-drop-behaviour-changing-between-helixtoolkit-and-helixtoolkitwpf/

A lot of DragEnter/DragLeave events when dragging over HelixViewport3D and its children.

### CODE REVIEW

File: Source/HelixToolkit.Wpf/Controls/HelixViewport3D.cs [hunk 1]
```diff
@@ -3224,6 +3224,7 @@ public override void OnApplyTemplate()
                 if (this.adornerLayer != null)
                 {
                     this.adornerLayer.Child = this.viewport;
+                    adornerLayer.IsHitTestVisible = false;
```
Comment (objorke) [inline Source/HelixToolkit.Wpf/Controls/HelixViewport3D.cs]: no side effects from this change?
I guess the adorner layer never needs hit testing.
Maybe it should be set in xaml instead: https://github.com/helix-toolkit/helix-toolkit/blob/master/Source/HelixToolkit.Wpf/Themes/Generic.xaml#L94

Comment (objorke) [inline Source/HelixToolkit.Wpf/Controls/HelixViewport3D.cs]: you forgot "this." - please follow style of existing code ;-)

**Comment by objorke** [comment]
I moved the code into the control template #253.
Does that work?

**Comment by objorke** [comment]
I merged #253. Please create an issue if it does not work.

**Comment by isukces** [comment]
Yes, drag&drop works properly. Thank you.

**Comment by desgriffiths** [comment]
I think this has the side-effect that the manipulator example no longer works.

**Comment by objorke** [comment]
@desgriffiths thanks for the notice! You are right, this PR broke a lot of other examples!

**Comment by objorke** [comment]
I applied a fix

**Pull request closed without merging**